doc/styles/config/vocabularies/ANSYS/accept.txt

```diff
@@ -1,8 +1,8 @@
-(?!)amsys
+(?i)ansys
+(?i)github
+(?)APIs
```

Comment:
@Revathyvenugopal162  This line is missing an operator. Did you mean to use (?i)APIs ?

Reply:
ye, good catch